audittools/doc.go

```diff
@@ -60,20 +63,19 @@ One usage of the aforementioned implementation can be:
 		}
 
 		rabbitmqQueueName := "down-the-rabbit-hole"
-		rabbitmqURI := amqp.URI{
```

Comment:
On second thought, let's not suggest `amqp.URI{}` in the docs, since most people will copy the example as is and end with up with an extra dependency like you.